src/pages/news/2024-09-20-tbtc-on-aave-proposal-passes-a-milestone-for-permissionless-money.md

```diff
@@ -0,0 +1,38 @@
+---
+template: news-item
+title: "tBTC on Aave Proposal Passes: A Milestone for Permissionless Money"
+description: Aave’s liquidity pools are some of the most liquid and widely used
+  in the entire DeFi space, meaning that tBTC holders will now have access to an
+  array of lending and borrowing opportunities previously unavailable.
+canonicalUrl: tbtc-on-aave
```

Comment:
This is wrong if it's meant to be this file, and we also need to canonicalize the Thesis blog post version or we'll tax our SEO juice.